src/ApiService/ApiService/onefuzzlib/Events.cs

```diff
@@ -60,15 +64,42 @@ public async Task SendEvent(BaseEvent anEvent)
 
         public void LogEvent(BaseEvent anEvent, EventType eventType)
         {
-            //todo
-            //var scrubedEvent = FilterEvent(anEvent);
-            //throw new NotImplementedException();
+            var options = EntityConverter.GetJsonSerializerOptions();
+            options.DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull;
+            options.Converters.Add(new RemoveUserInfo());
+            var serializedEvent = JsonSerializer.Serialize(anEvent, options);
+            _logger.LogInformation($"sending event: {eventType} - {serializedEvent}");
```

Comment:
using the built-in logger does not log info that we need to have. Correlation ID is not there, custom tags are not there. There is a ton of useful info that we should have and we cannot have due to way built in logger works..

Reply:
The version with the LogTracer was not logging when i ran the code locally so reverted that change. 
will bring it back when that the local scenario works